pkg/gadgets/dns/types/dns.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
+)
+
+type Event struct {
+	eventtypes.Event
+
+	DNSName string `json:"dnsName,omitempty"`
```

Comment:
Should we use `dnsName` or `dns_name`?. Same for pktType.

Also, should we append `kubernetes` to `namespace` and `pod`?

Reply:
I agree we should use the same snake_case format everywhere. Same for `kubernetes_` prefix. At least on process and socket collector we have it so I think it should be added for DNS, even though I just realised we missed it from PR #235.

Reply:
> Should we use dnsName or dns_name?. Same for pktType.

It needs to be exported because it's used outside of the package in cmd/kubectl-gadget/dns.go

> Also, should we append kubernetes to namespace and pod?

It should already be there from `eventtypes.Event` line 22.

Reply:
Oh, I misunderstood the comment. There does not seem to be a standard globally accepted but:
* Kubernetes uses `camelCase`
* The [Google JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format) says "Property names must be camel-cased, ascii strings"

Reply:
I think we must stick to the same convention in Inspektor Gadget. I told @blanquicet to use snake case on the process collector because I thought we're using that everywhere. (out extension to bcc tools uses snake_case https://github.com/kinvolk/bcc/commit/9e3be8bcf08f8b5ce07620db4179b94c71c2663c). I'm happy to switch to whatever style makes more sense for us.

Reply:
I agree. Either way, I guess this PR should not change the style everywhere, so I could update this code to `dns_name` and `pkt_type` to be consistent with the collectors and bcc tools, and we can see after if we change or not.